src/Print.hs

```diff
@@ -26,7 +26,7 @@ prettyBehaviour (Behaviour name contract interface preconditions cases postcondi
     prettyPre p = header "iff" >-< block (prettyExp <$> p)
 
     prettyCases [] = ""
-    prettyCases p = header "case" >-< block (prettyExp <$> p)
+    prettyCases p = header "case" >-< block (prettyExp <$> p) <> ":"
```

Comment:
The colon was missing and it was giving a parse error when trying to parse pretty printed code.